FpsCheck.py

```diff
@@ -83,9 +83,9 @@ def load_sets(self):
         load_sets_errors = []
         for fpset in self.fps_sites['sets']:
             primary = fpset.get('primary', None)
-            ccTLDs = fpset.get('ccTLDs', None)
-            associated_sites = fpset.get('associatedSites', None)
-            service_sites = fpset.get('serviceSites', None)
+            ccTLDs = fpset.get('ccTLDs', {})
+            associated_sites = fpset.get('associatedSites', [])
+            service_sites = fpset.get('serviceSites', [])
```

Comment:
IMO the defaults should not be handled here; they should be handled more centrally, in the `FpsSet` constructor.

```suggestion
            ccTLDs = fpset.get('ccTLDs')
            associated_sites = fpset.get('associatedSites')
            service_sites = fpset.get('serviceSites')
```